common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1096,6 +1096,79 @@
     ],
     "sqlState" : "38000"
   },
+  "FAILED_JDBC" : {
```

Comment:
`FAILED_JDBC` -> `FAILED_JDBC_OPERATION`

Reply:
Too long, does `_OPERATION` bring any benefits?

Reply:
Just because `FAILED_JDBC` looks confused.

Reply:
JDBC means Java Database **Connectivity**, the connectivity might fails at any time, ops or while establishing connections or while keeping it. For instance, this is real example:
```
org.postgresql.util.PSQLException: The connection attempt failed.
at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:331)
at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:223)
```

Reply:
I guess the connectivity is not changed if `create index failed`